pkg/manager/member/pd_member_manager.go

```diff
@@ -239,6 +252,58 @@ func (pmm *pdMemberManager) syncPDStatefulSetForTidbCluster(tc *v1alpha1.TidbClu
 
 	return pmm.updateStatefulSet(tc, newPDSet, oldPDSet)
 }
+
+func (pmm *pdMemberManager) syncPDClientCerts(tc *v1alpha1.TidbCluster) error {
+	ns := tc.GetNamespace()
+	tcName := tc.GetName()
+	commonName := fmt.Sprintf("%s-pd-client", tcName)
+
+	hostList := []string{
+		commonName,
+	}
+
+	certOpts := &controller.TiDBClusterCertOptions{
+		Namespace:  ns,
+		Instance:   tcName,
+		CommonName: commonName,
+		HostList:   hostList,
+		Component:  "pd",
+		Suffix:     "pd-client",
+	}
+
+	return pmm.certControl.Create(certOpts)
```

Comment:
this will create the client cert every loop

Reply:
`syncPDServerCerts()` and `syncPDClientCerts()` are only called when the StatefulSet doesn't exist (line 188)